CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## Unreleased
+
+### Added
+
+- `helmet.crossOriginEmbedderPolicy`: enabled by default
+- `helmet.crossOriginOpenerPolicy`: enabled by default
+- `helmet.crossOriginResourcePolicy`: enabled by default
+- `helmet.originAgentCluster`: enabled by default
+- Updated tests in `index.test.ts` to reflect the above middlewares being enabled by default
```

Comment:
This doesn't affect users. Could you remove this line?

Reply:
No problem, removed in [7eb78da](https://github.com/helmetjs/helmet/pull/317/commits/7eb78d1e61ac6fb8a91f99e22d82289317d232de).